upstash/assets/dashboards/upstash_redis_overview.json

```diff
@@ -0,0 +1,1008 @@
+{
+  "title": "Upstash Redis - Overview",
```

Comment:
This new dashboard needs to be added to the `manifest.json`. That is why the validations are failing :)